components/brave_rewards/browser/rewards_database.cc

```diff
@@ -0,0 +1,319 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+#include <utility>
+#include <string>
+#include <vector>
+
+#include "base/bind.h"
+#include "base/files/file_util.h"
+#include "brave/components/brave_rewards/browser/rewards_database.h"
+#include "sql/statement.h"
+#include "sql/transaction.h"
+
+namespace brave_rewards {
+
+namespace {
+
+void HandleBinding(
+    sql::Statement* statement,
+    const ledger::DBCommandBinding& binding) {
+  if (!statement) {
+    return;
```

Comment:
Wouldn't we want to return an error in this instance?  Or theoretically a DCHECK would suffice, as we'd only want to concern ourselves with a failed binding during dev?